File: DienTich.java
package application;

import java.util.Scanner;

public class DienTich {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Nhập chiều dài: ");
        double chieuDai = scanner.nextDouble();

        System.out.print("Nhập chiều rộng: ");
        double chieuRong = scanner.nextDouble();

        double dienTich = chieuDai * chieuRong;
        System.out.println("Diện tích hình chữ nhật là: " + dienTich);

        scanner.close();
    }
}

File: DienTich.js
function tinhDienTich() {
    const dai = parseFloat(document.getElementById("chieuDai").value);
    const rong = parseFloat(document.getElementById("chieuRong").value);
    const ketQua = document.getElementById("ketQua");

    if (isNaN(dai) || isNaN(rong) || dai <= 0 || rong <= 0) {
        ketQua.innerText = "Vui lòng nhập số hợp lệ cho chiều dài và chiều rộng.";
        ketQua.style.color = "red";
        return;
    }

    const dienTich = dai * rong;
    ketQua.innerText = `Diện tích hình chữ nhật là: ${dienTich.toFixed(2)}`;
    ketQua.style.color = "green";
}
